packages/revocation/src/implementations/revocationOnChain.ts

```diff
@@ -0,0 +1,127 @@
+import {
+  Contract, ethers, Event, utils,
+} from 'ethers';
+import { EwSigner, addressOf } from '@ew-did-registry/did-ethr-resolver';
+import { abi as RevocationOnChainAbi } from '../constants/RevocationRegistryOnChain.json';
```

Comment:
I think it would be to include iam-contracts as a dependency and then could import the contract json from iam-contracts directly.

Reply:
As discussed, it's better to have this change once the Typescript version is updated and relevant changes are done.